schema/m3-region.schema.json

```diff
@@ -537,6 +537,74 @@
                 }
               }
             }
+          },
+          "enemies": {
+            "$id": "#/items/properties/enemies",
+            "type": "array",
+            "title": "Enemies; Things that try to kill you.",
+            "items": {
+              "$id": "#/items/properties/enemies/items",
+              "type": "object",
+              "title": "Enemies",
+              "required": [
+                "name",
+                "quantity",
+                "homeNodes"
+              ],
+              "properties": {
+                "name": {
+                  "$id": "#/items/properties/enemies/items/properties/name",
+                  "type": "string",
+                  "title": "Enemy name",
+                  "default": "",
+                  "examples": [
+                    "Kihunter (green)"
+                  ]
+                },
+                "quantity": {
+                  "$id": "#/items/properties/enemies/items/properties/quantity",
+                  "type": "integer", 
+                  "title": "Quantity",
+                  "default": "",
+                  "examples": [
+                    3
+                  ]
+                },
+                "homeNodes": {
+                  "$id": "#/items/properties/enemies/items/properties/homeNodes",
+                  "type": [ "null", "array" ],
+                  "items": {
+                    "type": "integer"
+                  },
+                  "title": "Home Nodes",
+                  "description": "At what nodes enemies normaly engage you",
+                  "default": "",
+                  "examples": [
+                    [1, 2]
+                  ]
+                },
+                "betweenNodes": {
```

Comment:
I think it only makes sense to have 2 elements in this array? We can specify this with the following properties:
"minItems": 2,
"maxItems": 2,